components/toast_wrapper/toast_wrapper.tsx

```diff
@@ -396,12 +389,12 @@ class ToastWrapper extends React.PureComponent {
             width,
             onDismiss: this.hideUnreadWithBottomStartToast,
             onClick: this.scrollToUnreadMessages,
-            onClickMessage: Utils.localizeMessage('postlist.toast.scrollToUnread', 'Jump to unreads'),
+            onClickMessage: localizeMessage('postlist.toast.scrollToUnread', 'Jump to unreads'),
             showActions: true,
-            jumpDirection: 'up',
+            jumpDirection: 'up' as const,
```

Comment:
where is this type coming from?

Reply:
here you go 
https://github.com/mattermost/mattermost-webapp/blob/f3f4942debdf4db1ddb54605ccf103b3af320f47/components/toast/toast.tsx#L28